.travis.yml

```diff
@@ -1,17 +1,18 @@
 language:
   - cpp
 
+git:
+  depth: 1
+
 env:
   matrix:
-    # use libtorrent 0.15.10
-    # uncomment when Travis doesn't use Ubuntu 12.04 LTS, which has libtorrent 0.15.10 as package
-    #- lt_source=from_dist gui=true
-    #- lt_source=from_dist gui=false
-    # use libtorrent 0.16.X from RC_0_16 svn branch
-    - lt_source=from_svnRC_0_16 gui=true
-    - lt_source=from_svnRC_0_16 gui=false
-    - lt_source=from_svnRC_1_0 gui=true
-    - lt_source=from_svnRC_1_0 gui=false
+    # Uncomment when Travis doesn't use Ubuntu 12.04 LTS whose repo has more up-to-date libtorrent package
```

Comment:
You didn't write what I left in the comment. The current sentence doesn't make sense. It should be "repo will have a more up-to-date..."